open-grid-scheduler/Vagrantfile

```diff
@@ -84,6 +84,27 @@ Vagrant.configure(VAGRANTFILE_API) do |config|
             "head"    => [head_name],
             "workers" => (1..N).map { |w| get_worker_name(w) }
           }
+          # Extra variables for extra nfs_exports, extra users and extra submit hosts
+          #ansible.extra_vars = {
+          #  nfs_extra_exports: [
+          #    {
+          #      path: "/exports",
+          #      options: "*(rw,sync,no_subtree_check,no_root_squash)"
+          #    }
+          #  ],
+          #  ogs_external_submit_hosts:[
+          #    {
+          #      hostname: "external_host",
+          #      ip_address: "10.0.0.10"
+          #    }
+          #  ],
+          #  ogs_extra_users:[
+          #    {
+          #      uid: "1300",
+          #      name: "extra"
+          #    }
+          #  ],
+          #}
```

Comment:
Just a little formatting issue ~ I think I'd rather have the `ansible_extra_vars` uncommented but left empty and then the available flags commented.

Something like:
```
ansible_extra_vars = {}
# Available options:
# nfs_extra_exports: [
#   {
#     path: "/exports" 
#     options: ""
#   }
# ]

and so on ..
```